virt/virtual_machines/advanced_vm_management/virt-configuring-a-watchdog.adoc

```diff
@@ -0,0 +1,15 @@
+[id="virt-configuring-a-watchdog"]
+= Configuring a watchdog
+include::modules/virt-document-attributes.adoc[]
+:context: virt-configuring-a-watchdog
+
+toc::[]
+
+Expose a watchdog device on a virtual machine instance (VMI), alongside liveness and readiness probes, to ensure the operating system is running properly.
+
```

Comment:
I'm not sure that this sentence describes the modules. Are the liveness and readiness probes mentioned somewhere?

I think the user would expect something like this. 

Suggestion: You can configure a watchdog by configuring the VM CR for a watchdog device, installing watchdog, and starting the watchdog service.

That would summarize the two modules and explain their order.